.github/workflows/ci.yml

```
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [16.x, 15.x, 14.x, 13.x, 12.x, 10.x, "10.12.0"]
+        node: [16.x, 15.x, 14.x, 12.x]
```

We typically keep one build in the matrix pinned to our lowest supported version. With Node 12 now in maintenance, it's less likely we'll accidentally rely on something changed in >12.22.0, but it's nice to know for sure.

```suggestion
        node: [16.x, 15.x, 14.x, 12.x, "12.22.0"]
```